protocol-units/bridge/chains/movement/src/lib.rs

```diff
@@ -458,12 +459,12 @@ impl BridgeContractCounterparty for MovementClient {
 	) -> BridgeContractCounterpartyResult<()> {
 
 		let args = vec![
-			utils::serialize_vec(&initiator.0)?,
-			utils::serialize_vec(&bridge_transfer_id.0[..])?,
-			utils::serialize_vec(&hash_lock.0[..])?,
-			utils::serialize_u64(&time_lock.0)?,
-			utils::serialize_vec(&recipient.0.0.to_vec())?,
-			utils::serialize_u64(&amount.0)?,
+			to_bcs_bytes(&initiator.0).unwrap(),
```

Comment:
I think you resolved the conflict incorrectly, use the new utils:: methods created that bubble up the error with ? Don't unwrap. 

These were added by @andygolay in https://github.com/movementlabsxyz/movement/pull/215/commits/3cde18ff89de6330d620b3a0de7676632ea1caa4

Reply:
conflict was around line 406, not this one, let me take a look